app/javascript/components/track/iteration-summary/TestsStatusSummary.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { TestRunStatus } from '../../student/Editor'
+
+function Content({ testsStatus }: { testsStatus: TestRunStatus }) {
+  switch (testsStatus) {
+    case TestRunStatus.QUEUED:
+      return (
+        <>
+          <svg role="presentation" className="icon">
+            <use xlinkHref="#loading" />
+          </svg>
+          <div className="status">Queued</div>
+        </>
+      )
+    case TestRunStatus.PASS:
+      return (
+        <>
+          <div className="dot"></div>
+          <div className="status">Passed</div>
```

Comment:
```suggestion
          <div className="status"  id="iteration-test-run-status">Passed</div>
```

Reply:
@SleeplessByte Why the `id` here? Is this component guaranteed to be unique on the page?

Reply:
See: https://github.com/exercism/v3-website/pull/224#discussion_r514410556